File: Assignment_35/Program2.java
import java.util.*;

class StringX
{
    public static int CountSamll(String str)
    {
        int icnt = 0;
        int i = 0;
        for(i = 0; i<str.length(); i++)
        {
            char ch = str.charAt(i);

            if(ch >= 'a' && ch  <= 'z')
            {
                icnt++;
            }
        }
        return icnt;
    }
}
class Program2
{
    public static void main(String[] args) 
    {
        int iRet = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter String :");
        String input = sobj.nextLine();

        StringX xobj = new StringX();

        iRet = xobj.CountSamll(input);

        System.out.println("Count is :"+iRet);

         
    }
}

File: Assignment_35/Program2.c
#include<stdio.h>

int CountSmall(char *str)
{
    int icnt = 0;
    while(*str != '\0')
    {
        if(*str >= 'a' && *str  <= 'z')
        {
            icnt++; 
        }
        str++;
    }
    return icnt;
}

int main()
{
    char Arr[20];
    int iRet = 0;

    printf("Enter String :\n");
    scanf("%[^'\n']s",Arr);

    iRet = CountSmall(Arr);

    printf("%d",iRet);

    return 0;
}